graylog2-web-interface/src/components/alertnotifications/AlertNotification.jsx

```diff
@@ -18,8 +17,16 @@ const AlertNotification = React.createClass({
   },
   mixins: [Reflux.connect(AlertNotificationsStore)],
 
+  getInitialState() {
+    return {
+      isTestingAlert: false,
+    };
+  },
+
   _onTestNotification() {
-    StreamsStore.sendDummyAlert(this.props.alertNotification.stream_id);
```

Comment:
I was wrong, we use it in some other old components that I didn't clean up yet. As we have another task in #3178 to clean up the code, I would postpone the removal until then.